ansible/playbooks/qa_framework.yml

```diff
@@ -0,0 +1,12 @@
+- hosts: qa_framework
+  roles:
+    - ../roles/qa_framework
+  vars:
+    qa_hosts:
+      - port: 1514
+        protocol: tcp
+        api_port: 55000
+        api_proto: 'http'
+        api_user: ansible
+        max_retries: 5
+        retry_interval: 5
```

Comment:
This is not required

Reply:
Solved https://github.com/wazuh/wazuh-qa/pull/3032/commits/30c9ae7e7f5b647b6eacc980bb1274fc06d9a077